tensorflow/stream_executor/cuda/cuda_driver.cc

```diff
@@ -308,9 +308,12 @@ static port::Status InternalInit() {
 
   if (res == CUDA_SUCCESS) {
     return port::Status::OK();
+  } else if (res == CUDA_ERROR_SHARED_OBJECT_INIT_FAILED) {
+    LOG(WARNING) << "failed call to cuInit: " << ToString(res);    
```

Comment:
@chisgg Just to be clear `ToString(res)` as it was designed will be always classify also this case as `Unkown Error` for the LOG string but we really know that we are in `cudaErrorSharedObjectInitFailed = 303 This indicates that initialization of a shared object failed. `